### PR TITLE
Fixed Overflow of Navbar.

### LINKED ISSUE #4: Fixed Overflow in Navbar
Bug Screenshot

### PR DESCRIPTION
**Pull request by Phoenix6296**
Fixes #4  Added some media queries in the style.css for fixing the overflow problem of Navbar.

### CODE REVIEW

**Comment by Mitrajit** [comment]
The Sharenetic title seems even smaller than the buttons. Let's do something else to make the proportions correct.
The download and upload icons seem too spaced.
image
TODO
- [x] Correct the 'Sharenetic' title proportions
- [x] Reduce the spaces between upload and download icon
- [x] Also try Samsung fold and try to correct the overflow

File: index.html [hunk 1]
```diff
@@ -18,30 +18,38 @@
...
     </div>
   </div>
-  <div class="container box p-4">
-    <div class="input-group pb-3">
-      <input type="text" class="form-control" placeholder="Recipient's ID" aria-label="Recipient's ID"
-        aria-describedby="button-addon2" id="Recipient">
+  <div class="container box p-4 whole-body">
+    <div class="input-group pb-3 head-place">
+      <div class="recipent">
+        <input type="text" class="form-control" id="recipent-input" placeholder="Recipient's ID"
```
Comment (Mitrajit) [inline index.html]: The ID for the textbox was changed. Please don't change any ID as they are crucial for the javascript to work.

**Review by Mitrajit — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Phoenix6296) [reply index.html]: Done Dona Done :)

**Comment by Mitrajit** [comment]
Thanks 👍🏻

**Pull request merged**